core/lib/spree/core/controller_helpers/auth.rb

```diff
@@ -63,9 +63,10 @@ def try_spree_current_user
         # Redirect as appropriate when an access request fails.  The default action is to redirect to the login screen.
         # Override this method in your controllers if you want to have special behavior in case the user is not authorized
         # to access the requested action.  For example, a popup window might simply close itself.
-        def redirect_unauthorized_access
+        def redirect_unauthorized_access(exception)
```

Comment:
Assignment Branch Condition size for redirect_unauthorized_access is too high. [22.2/15]